src/js/formats/twoSided.js

```diff
@@ -0,0 +1,42 @@
+import twoSidedTemplate from '../text/twoSided.dot.html!text';
+import slider from '../lib/slider';
+import q from '../lib/query';
+
+export default {
+    preprocess(data) {
+        return data;
+    },
+    postRender(data) {
+        function goToAnswer(answer) {
```

Comment:
What is the boundary between the content of the `postRender` function and the content of the `slider` library? It looks like they both share the responsibility for defining a two-sided atom.